### PR TITLE
Build for osx-arm64

### PR DESCRIPTION
**Pull request by vasudevanv**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,13 +10,15 @@ source:
...
-  number: 1
+  number: 2
   script: {{ PYTHON }} -m pip install . -vv
 
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
+    - python
+    - numpy
```
Comment (jan-janssen) [inline recipe/meta.yaml:21]:
```suggestion
    - cross-python_{{ target_platform }}    # [build_platform != target_platform]
    - python                                # [build_platform != target_platform]
    - numpy                                 # [build_platform != target_platform]
```

**Comment by jan-janssen** [comment]
I opened the corresponding request on https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/4354

**Comment by jan-janssen** [comment]
Closing this one in favor of https://github.com/conda-forge/scikit-fmm-feedstock/pull/29

**Comment by vasudevanv** [comment]
Thank you so much!

**Pull request closed without merging**